docs/cordova.md

```diff
@@ -1,25 +1,26 @@
-Usage in Cordova apps
-=====================
+# Usage in Cordova apps
 
-`Apache Cordova  <https://cordova.apache.org>`_ is a mobile development
+[Apache Cordova](https://cordova.apache.org) is a mobile development
 framework. It allows you to use standard web technologies - HTML5, CSS3,
 and JavaScript for cross-platform development. Applications execute
 within wrappers targeted to each platform, and rely on
-standards-compliant API bindings to access each device's capabilities
-such as sensors, data, network status, etc. [#f1]_
+standards-compliant API bindings to access each device\'s capabilities
+such as sensors, data, network status, etc.
 
-To use remoteStorage.js in a Cordova app, you need to have the `InAppBrowser plugin
-<https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-inappbrowser/>`_
+To use remoteStorage.js in a Cordova app, you need to have the
+[InAppBrowser
```

Comment:
Does this line break need to be removed?

Reply:
No, it's in line with the width constraints of the rest of the text.